connectors/redis/redis_test.go

```diff
@@ -164,9 +161,6 @@ func TestReadNotFound(t *testing.T) {
 		t.Skip("Redis is not running")
 	}
 
-	table, _ := dosa.TableFromInstance(&testentity.KeyValue{})
```

Comment:
In the future please pull refactoring changes like this out into their own PR. it makes things easier to review.